api/src/main/java/ai/djl/translate/ServingTranslatorFactory.java

```diff
@@ -98,10 +99,17 @@ private ServingTranslator findTranslator(Path path, String className) {
             Path classesDir = path.resolve("classes");
             compileJavaClass(classesDir);
 
-            List<Path> jarFiles =
-                    Files.list(path)
-                            .filter(p -> p.toString().endsWith(".jar"))
-                            .collect(Collectors.toList());
+            List<Path> jarFiles = new ArrayList<>();
+            if (Files.isDirectory(path)) {
```

Comment:
If the folder doesn't exists, IOException will throw, this will prevent class loading from system classpath